modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.rules_defaults.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Provide default rules integration to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_default_rules_configuration().
+ */
+function fb_instant_articles_api_rules_default_rules_configuration() {
+  // Handling add/update node article as published in FB Instant Articles library.
+  $rule = rules_reaction_rule();
+  $rule->label = t('Add/update as published in FB Instant Articles library ' .
+                   'when a published Article node is saved');
+  $rule->active = FALSE;
+  $rule->event('node_presave')
+    ->condition('data_is', array('data:select' => 'node:status', 'value' => TRUE))
+    ->condition('data_is', array('data:select' => 'node:type', 'value' => 'article'))
+    ->action('fb_instant_articles_api_rules_action_import', array('data:select' => 'node'));
+
+  $configs['fb_instant_articles_rules_default_published'] = $rule;
+
+  // Handling add/update node article as unpublished in FB Instant Articles library.
+  $rule = rules_reaction_rule();
+  $rule->label = t('Add/update as unpublished in FB Instant Articles library ' .
+    'when a unpublished Article node is saved');
+  $rule->active = FALSE;
+  $rule->event('node_presave')
+    ->condition('data_is', array('data:select' => 'node:status', 'value' => FALSE))
```

Comment:
Line could be replaced with:

`->condition(rules_condition('node_is_published')->negate())`

(seems a little cleaner to me)